src/main/scala/vrp/Vrp.scala

```diff
@@ -0,0 +1,31 @@
+package vrp
```

Comment:
The files in this package are mostly copied and adjusted to VRP. Can't we have partly the same model / reuse some of the code?